src/hooks/cakeVault/useGetVaultUserInfo.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react'
+import BigNumber from 'bignumber.js'
+import { useWeb3React } from '@web3-react/core'
+import { useCakeVaultContract } from 'hooks/useContract'
+
+const useGetVaultUserInfo = (lastUpdated?: number) => {
```

Comment:
Can we put return types in functions? Apart form maybe some simple `number` or `string`, although I'd personally prefer even them to be explicitly defined.
I assume this returns VaultUser that defined in types, but would be nice not to jump around to verify that. Wastes extra 2-3 seconds but it adds up.

Reply:
Typed the state here, I think that'll now get inferred in the return values too